app/jobs/consume_assessments_job.rb

```diff
@@ -41,6 +42,15 @@ def perform
           next
         end
 
+        # Error occured in twilio studio flow
+        unless message['error_code'].blank?
+          TwilioSender.handle_twilio_error_codes(patient, message['error_code'])
+          # Will attempt to resend assessment if phone is off or if unkown error occurs
+          patient.update(last_assessment_reminder_sent: nil) if message['error_code']&.in? %w[30003 30008]
```

Comment:
Something I would like reviewers to think about is if this group of error codes `30003 and 30008` (see TwilioSender for details about these error codes) that we would like to set `last_assessment_reminder_sent` to nil for (indicating a retry if still in the contact period).

Reply:
Two things:

1. I think we should put the list of these error codes in a constant or variable somewhere with a comment.
2. We should discuss including error code 30003 in this, as we see upwards of 1.5K of these on prod and that'd be a fair number of retries if the majority are just landlines, for example, and shouldn't even be retried for SMS.

@holmesie thoughts?

Reply:
Agree 100% with both of those points

Reply:
Want to close the loop on this discussion. It's now set to retry on 30003 only. Why did we decide not to retry on 30008 (unknown error)? Was there any offline discussion about # 2 in Kylie's comment above?

Reply:
Good question. IIRC the reason for the removal of :"unkown error" as a retry eligible code had to do with certainty of the cause and chances that the retry will not result in the same error. I do think that there could certainly be further discussion on which codes should be included in the retry_eligible set, especially since there are ~1k 30003s a day some of which **could** be have been caused not by sending an SMS to a cellphone that was off but by sending an SMS to a landline depending on how the carrier reported the error.

Reply:
Alright well I don't think I know enough about all the possible causes of each error to debate this, so I'm content with only including 30003 now. With the groundwork laid by this PR, it will be easy to add in the future if we want to.